pkg/docker/containers-view.jsx

```diff
@@ -32,6 +32,20 @@ var Listing = require('cockpit-components-listing.jsx');
 var Select = require('cockpit-components-select.jsx');
 var moment = require('moment');
 
+var problems_client = cockpit.dbus('org.freedesktop.problems', { superuser: "try" });
```

Comment:
Should this maybe happen in a create function so that the client can be closed in a destroy function?

Reply:
Can you explain this in a bit more detail?

Reply:
@petervo Updated